### PR TITLE
[forge-viewer] update for version 7.58

### PR DESCRIPTION
**Pull request by liskaj**
- updated ToolInterface
- updated Viewer3D
- updated DataVisualization
- updated Box2
- updated Box3
- updated ExtrudeGeometry
- updated AggregatedViewOptions
- updated Model

### CODE REVIEW

File: types/forge-viewer/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for non-npm package Forge Viewer 7.55
+// Type definitions for non-npm package Forge Viewer 7.58
 // Project: https://forge.autodesk.com/en/docs/viewer/v7/reference/javascript/viewer3d/
```
Comment (domdomegg) [inline types/forge-viewer/index.d.ts:2]:
```suggestion
// Project: https://forge.autodesk.com/en/docs/viewer/v7/developers_guide/overview/
```

Link broken

File: types/forge-viewer/index.d.ts [hunk 2]
```diff
@@ -939,27 +940,27 @@ declare namespace Autodesk {
         }
 
         class ToolInterface {
```
Comment (domdomegg) [inline types/forge-viewer/index.d.ts:942]: I'm not sure I see where where this is changed?

https://forge.autodesk.com/en/docs/viewer/v7/reference/Viewing/ToolInterface/

Comment (liskaj) [reply types/forge-viewer/index.d.ts:942]: Some methods of the class are optional.

**Review by domdomegg — APPROVED** [review]

**Review by domdomegg — APPROVED** [review]

**Comment by liskaj** [comment]
Ready to merge

**Pull request merged**